PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/model/AzureSdkArtifactEntity.java

```diff
@@ -5,41 +5,42 @@
 
 package com.microsoft.azure.toolkit.intellij.azuresdk.model;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import lombok.ToString;
+import lombok.Setter;
+
+import java.util.Map;
 
-@ToString
 @Getter
+@Setter
 @NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class AzureSdkArtifactEntity {
-    @JsonProperty("Package")
-    private String packageName;
-    @JsonProperty("GroupId")
+    private String artifactId;
```

Comment:
It seem that the raw data also provide compatibility data for libraries, is this a backlog feature?

Reply:
the full feature actually has 2 sub features:
* sdk reference book(readonly)
* dependency management

dependency compatibility is part of dependency management, which is not planned in this sprint.